frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx

```diff
@@ -117,11 +118,19 @@ export const IndividualResponsePage = (): JSX.Element => {
   ])
 
   const backLink = useMemo(() => {
+    const searchParams = new URLSearchParams()
+    if (!lastNavPage && !lastNavSubmissionId) {
+      return `..`
+    }
+
     if (lastNavPage) {
-      return `..?page=${lastNavPage}`
+      searchParams.append('page', lastNavPage.toString())
+    }
+    if (lastNavSubmissionId) {
+      searchParams.append('submissionId', lastNavSubmissionId)
```

Comment:
/nit Use `set()` rather than append, to show this has nothing to do with multi-value args.

```suggestion
      searchParams.set('page', lastNavPage.toString())
    }
    if (lastNavSubmissionId) {
      searchParams.set('submissionId', lastNavSubmissionId)
```

Reply:
this does have something to do with multivalue args (but also not really since one response search can only have one result, but does future proof if we ever allow partial id searching in the future)

Reply:
Sorry, what does partial id searching mean in this context?

`.append()` is used if you want to have 2 or more `pages` key  in the query string, or 2 or more `submissionId` in the query string. I assume we will not have that, and so `.set()` should be used instead.

Of course functionally it doesn't matter here, since the code starts with a fresh `URLSearchParams` anyway, but semantically it's clearer to use `.set()` to indicate intent.

```javascript
p = new URLSearchParams()
p.set('foo', 1)
p.set('bar', 2)

p.toString() // foo=1&bar=2

p.append('foo', 3)

p.toString() // foo=1&bar=2&foo=3

p.set('foo', 4)

p.toString() // foo=4&bar=2
```

Personally, I'm really not fan of multi-value keys in query string btw (e.g. `foo=1&foo=2`), I much prefer having a singular list key that includes multiple entries with some encode/parse convention, like `fooList=1,2`

Reply:
fuzzy search, but i'll just reset it as you suggested. We can reimplement multi-arg in the future if needed.

Reply:
ah i got you. yes, we should use `set` instead of `append`. that was a bug!!!

Reply:
fixed in f3e05e5